.eslintignore

```diff
@@ -5,4 +5,5 @@ node_modules
 packages/*/*/doc
 packages/*/*/index.ts
 packages/**/bad.js
+packages/externals/create-electron-app/src/*
```

Comment:
Just a callout - this is added because eslint doesn't allow shebangs or requires (which is essentially the only lines in the JS file)